smartclide-monitoring/src/main/java/eu/smartclide/contexthandling/services/GitlabApiClient.java

```diff
@@ -123,8 +123,15 @@ public List<GitlabCommitMessage> getGitlabCommitMessages() {
     private int calculateTimeSinceLastCommit(String projectId, JsonObject commit) {
         int difference = 0;
         // check if parent id exists for given commit
-        if (commit.get("parent_ids").getAsJsonArray().size() > 0) {
-            String parentCommitId = commit.get("parent_ids").getAsJsonArray().get(0).getAsString();
+        JsonArray parentIds = commit.get("parent_ids").getAsJsonArray();
+        if (parentIds.size() > 0) {
+            logger.info("-----------------Found {} parent_ids with commit id {}", parentIds.size(), commit.get("id").getAsString());
+            logger.info("parent_ids: {}", parentIds);
+
+            // consider first commitId from parentIds array
+            // because Gitlab API always provides commit from original branch as first element of the parent_ids array
+            String parentCommitId = parentIds.get(0).getAsString();
+            logger.info("parent_ids[0]: {}", parentCommitId);
```

Comment:
I think this log statement could be removed because the commit ids are already logged in line 129.